src/azul/plugins/metadata/hca/__init__.py

```diff
@@ -297,6 +298,9 @@ def special_fields(self) -> SpecialFields:
                              bundle_version='bundleVersion',
                              root_entity_id='projectId')
 
+    def modify_hits(self, hits: MutableJSONs) -> None:
+        return None
```

Comment:
```suggestion
        pass
```
Since the return value is not meant to be used